views/scss/qti/_qti-base.scss

```
@@ -270,51 +275,65 @@ $mainContainer : '.qti-item';
             cursor: default;
         }
 
-        input[type="color"],
-        input[type="date"],
-        input[type="datetime"],
-        input[type="datetime-local"],
-        input[type="email"],
-        input[type="month"],
-        input[type="number"],
-        input[type="range"],
-        input[type="search"],
-        input[type="tel"],
-        input[type="time"],
-        input[type="text"],
-        input[type="password"],
-        input[type="url"],
-        input[type="week"],
+        input[type='color'],
+        input[type='date'],
+        input[type='datetime'],
+        input[type='datetime-local'],
+        input[type='email'],
+        input[type='month'],
+        input[type='number'],
+        input[type='range'],
+        input[type='search'],
+        input[type='tel'],
+        input[type='time'],
+        input[type='text'],
+        input[type='password'],
+        input[type='url'],
+        input[type='week'],
         textarea,
         select {
-            border-top-color  : #aaa;
-            border-left-color : #aaa;
+            border-top-color: #aaa;
+            border-left-color: #aaa;
         }
     }
 
-    @import "all-interactions";
-    @import "object";
-    @import "choice";
-    @import "order";
-    @import "extended-text";
-    @import "feedback";
-    @import "associate";
-    @import "match";
-    @import "media";
-    @import "graphic";
-    @import "hottext";
-    @import "gap-match";
-    @import "slider";
-    @import "inline-choice";
-    @import "text-entry";
-    @import "upload";
-    @import "rubric-block";
-    @import "custom";
-    @import "table";
-    @import "tooltip";
-    @import "figure";
+    @import 'all-interactions';
+    @import 'object';
+    @import 'choice';
+    @import 'order';
+    @import 'extended-text';
+    @import 'feedback';
+    @import 'associate';
+    @import 'match';
+    @import 'media';
+    @import 'graphic';
+    @import 'hottext';
+    @import 'gap-match';
+    @import 'slider';
+    @import 'inline-choice';
+    @import 'text-entry';
+    @import 'upload';
+    @import 'rubric-block';
+    @import 'custom';
+    @import 'table';
+    @import 'tooltip';
+    @import 'figure';
 }
 
-.tao-preview-scope{
+.tao-preview-scope {
     padding-top: 20px;
 }
+
+.extendedText-shadow-caret {
+    position: absolute;
+    display: none;
+    width: 1.2em;
+    height: 1px;
+    background-color: $textColor;
+    animation: extendedText-shadow-caret-blink 750ms steps(4, start) infinite;
```

The animation doesn't look very natural to me. I tried the following modification, to match other browsers. Maybe your OS is giving a different result?
```suggestion
    animation: extendedText-shadow-caret-blink 1000ms steps(2, start) infinite;
```

---

Thanks! Yes, this feels more natural